design/FY2017/MultipleAirloopsInOneZone.md

```diff
@@ -0,0 +1,546 @@
+
+	*MJW 2. This would be good in general for the existing sizing and terminal unit objects.  I'll work on this table and that may help guide this proposal or at least provide some discussion points.  Revised Nov 14, 2016 - this is no longer needed, because the proposed new object is very simple with no overlapping fields.*
+
+10/31 - Gu
+
+1. You have an option: 6. Optional - Allow an airloop with no return path. How do you handle mass balance without return path? Do you plan to use other objects?
+
+	*MJW 1. The zone air balance would work as it currently does, except there wouldn't be a return node for that air loop.  The zone air mass balance should work the same as it currently does.  If there is more supply than exhaust (plus return to another airloop if present) then the excess air is assumed to exfiltrate and doesn't impact the zone air balance.  Of if the ZoneAirMassBalance option is active then that would take over and balance by adjusting infiltration and/or mixing.*
+
+	*For the airloop itself without return, it would need to take in outdoor air to match the supply and complain if the OA controller didn't allow that.*
+
+11/1 - Horowitz
+
+1. Allowing an airloop with no return path has been identified as a high priority for modeling airflow in residential buildings. If the funding allows for this, we would definitely like to see it included in the work.
+
+11/9 - Sizing Group Conference Call
+
+1. Try to minimize the size of the new DesignSpecification:AirTerminal:Sizing object.  Think in terms of only what might be different from the base Sizing:Zone inputs.  Fields such as VentilationRequirement, Do/Don't use OA specifications, 
+
+## Design ##
+### 1. Remove error checks on number of air terminals in a zone ###
+Actually, the only existing check looks for *an air distribution unit and an AirTerminal:SingleDuct:Uncontrolled (direct air) object in the same zone.* This check will be removed.  The code already allows more than one air distribution unit without throwing an error.
+
+#### ZoneEquipmentManager::SimZoneEquipment ####
+
+- Delete all uses of `ZoneHasAirLoopHVACTerminal` and `ZoneHasAirLoopHVACDirectAir` which includes the error check.  That's the only purpose of these variables.
+
+### 2. Allow more than one return air node in a zone ###
+
+#### DataZoneEquipment.hh ####
+In struct `EquipConfiguration`
+
+ - Delete `int ReturnAirNode`
+ - Add `int NumReturnNodes`
+ - Add `Array1D_int ReturnNode`
+ - Add to IDD and input processing for Return Air Node or Nodelist
+
+ - Delete `int AirLoopNum`
+ - Add `int NumAirLoops` (this may not always equal `NumReturnNodes` if an airloop has no return path)
+ - Add `Array1D_int AirLoopPointer`
```

Comment:
Would be nice if these lined up (i.e., each has same size as NumAirLoops). Remember the confusion with zone indexes for conditioned vs non-conditioned zones (if I recall correctly). This way, AirLoopPointer(6) always uses ReturnNode(6).